cmd/completion.go

```diff
@@ -57,6 +57,12 @@ var completionExample = `
 	# Set the dapr completion code for zsh[1] to autoload on startup
   	dapr completion zsh > "${fpath[1]}/_dapr"
 
+	# Installing fish completion on Linux
+	## Load the dapr completion code for fish into the current shell
+	dapr completion fish | source
+	# Set the dapr completion code for fish[1] to autoload on startup
+	dapr completion fish > $HOME/.config/fish/functions/dapr.fish
```

Comment:
We can do similar to line 49, i.e. first create completion file, and then `source ~/<file>` it.